packages/vscode-extension/src/extension.ts

```diff
@@ -437,6 +443,12 @@ export async function activate(context: vscode.ExtensionContext) {
     workspacePath && (await isSPFxProject(workspacePath))
   );
 
+  vscode.commands.executeCommand(
+    "setContext",
+    "fx-extension.isInitAppEnabled",
+    workspacePath && isInitAppEnabled()
```

Comment:
I think we shouldn't include workspacePath check here, right?

Reply:
Nice catch! Yes, no need to check workspace here.

Reply:
Updated, please take a look.